samples/subsys/usbc/sink/prj.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_USBC_THREAD_PRIORITY=10
```

Comment:
What about `CONFIG_USBC_TCPC_DRIVER`?

Reply:
This is enabled when the USBC stack is enabled.

menuconfig USBC_STACK
        bool "USBC Stack Support"
        default y
        depends on DT_HAS_USBC_PORT_ENABLED
        select SMF
        select SMF_ANCESTOR_SUPPORT
        select USBC_TCPC_DRIVER

Reply:
ok, right. But actually we should keep a high level Kconfig flag to allow user to enable the SW stack manually (just like we do for every subsystem or peripheral: ADC, SPI, USB_DEVICE_STACK, NETWORKING, ...).
So in this prj.conf, you should add `CONFIG_USBC_STACK=y`.
And `menuconfig USBC_STACK` should not use any `default y if foo` to avoid this level of control.

Reply:
I understand. Thanks